bin/fetch-configlet

```diff
@@ -2,7 +2,29 @@
 
 LATEST=https://github.com/exercism/configlet/releases/latest
 
+OS=$(
+case $(uname) in
+    Darwin*)
+        echo "mac";;
+    Linux*)
+        echo "linux";;
+    Windows*)
+        echo "windows";;
+    *)
+        echo "linux";;
+esac)
+
+ARCH=$(
+case $(uname -m) in
+    *64*)
+        echo 64bit;;
+    *32*)
```

Comment:
Are you sure about this one? I think `i386` is the classic response for a 32 bit architecture.